profiles/arch/sparc/32ul/package.mask

```diff
@@ -0,0 +1,7 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# matoro <matoro@users.noreply.github.com> (2022-08-05)
```

Comment:
Please use an email address people can actually reach you with.

Reply:
They can, despite the noreply in the name it will forward to my Github address.

Reply:
Nobody is going to end up trying because of the email address. Even if it's fine, it ends up looking antisocial, unfortunately (and reflects somewhat poorly on the person who merges in terms of due diligence).

Reply:
Changed it